src/resources/pages.json

```diff
@@ -18,12 +18,12 @@
   "newApplet": "/builder/new-applet",
   "newAppletAbout": "/builder/new-applet/about",
   "newAppletActivities": "/builder/new-applet/activities",
-  "newAppletNewActivity": "/builder/new-applet/activities/new-activity",
-  "newAppletNewActivityAbout": "/builder/new-applet/activities/new-activity/about",
-  "newAppletNewActivityItems": "/builder/new-applet/activities/new-activity/items",
-  "newAppletNewActivityItemFlow": "/builder/new-applet/activities/new-activity/item-flow",
-  "newAppletNewActivitySettings": "/builder/new-applet/activities/new-activity/settings",
-  "newAppletNewActivitySettingsItem": "/builder/new-applet/activities/new-activity/settings/:setting",
+  "newAppletActivity": "/builder/new-applet/activities/:activityId",
```

Comment:
pls add TODO to avoid usage of hardcoded `new-applet` path in favor of generatedPath from `new-applet` or ':appletId'
I suppose this file should completely removed afterwards

Reply:
done